yew-router/examples/servers/actix/src/main.rs

```diff
@@ -2,7 +2,7 @@ use actix_files::NamedFile;
 use actix_web::{get, middleware, web, App, Error, HttpResponse, HttpServer};
 
 // You will need to change this if you use this as a template for your application.
-const ASSETS_DIR: &str = "../../static";
+const ASSETS_DIR: &str = "../../_static";
```

Comment:
Reason I renamed this is to make it stand out; `static` when it's sorted alphabetically with the rest of the examples, looks like it's the name of an example.